src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2075,10 +2075,6 @@ private bool BuildProperties(IIndentedStringBuilder writer, XamlObjectDefinition
 								writer.AppendLineInvariant($"{setterPrefix}Color = {BuildColor(content)}");
 							}
 						}
-						else if (IsLinearGradientBrush(topLevelControl.Type))
```

Comment:
The method could be deleted if it's not used anywhere else.

Reply:
True that!

Reply:
Done